client/components/ConversationSearch/index.js

```diff
@@ -1,13 +1,33 @@
-import React from 'react';
+import React  from 'react';
+import axios from 'axios';
 import './ConversationSearch.css';
 
-export default function ConversationSearch() {
+export default function ConversationSearch({ setChat, chat }) {
+    
+  const getChat = async (search, chat) => {
+    return axios.get(`http://localhost:8000/api/chats/search/${search}`, {headers: { Authorization: `Bearer ${localStorage.auth}` }})
```

Comment:
we have "localhost" here. I think you might want to implement this with redux and use the production URL instead of localhost

Reply:
Done as requested

Reply:
Done as requested